backend/src/main/java/de/amos/apachepulsarui/dto/SubscriptionDto.java

```diff
@@ -8,24 +8,52 @@
 import lombok.AccessLevel;
 import lombok.Builder;
 import lombok.Data;
+import org.apache.pulsar.common.policies.data.SubscriptionStats;
 
 import java.util.List;
+import java.util.Objects;
 
 @Data
 @Builder(access = AccessLevel.PRIVATE)
 public class SubscriptionDto {
 
     private String name;
 
-    private List<ConsumerDto> consumers;
+    private ConsumerDto activeConsumer;
+
+    private List<ConsumerDto> allConsumers;
 
     private long numberConsumers;
 
-    public static SubscriptionDto createSubscriptionDto(String name, List<ConsumerDto> consumers) {
+    private double msgAckRate;
+
+    private long msgBacklog;
+
+    private long msgOutCounter;
+
+    public static SubscriptionDto createSubscriptionDto(SubscriptionStats subscriptionStats, String name) {
```

Comment:
Feels a bit verbose to me. Personally, I think `create()` is enough for a static factory method.